modules/javafx.controls/src/main/java/javafx/scene/control/skin/PaginationSkin.java

```diff
@@ -1338,8 +1356,9 @@ public int getPageNumber() {
         }
 
         public void release() {
-            getSkinnable().getStyleClass().removeListener(updateSkinIndicatorType);
-            tooltipVisibleProperty().removeListener(updateTooltipVisibility);
+            // FIX own listener helper?
+            //getSkinnable().getStyleClass().removeListener(updateSkinIndicatorType);
+            //tooltipVisibleProperty().removeListener(updateTooltipVisibility);
```

Comment:
I don't quite understand the comment. What follow-up is needed?

Reply:
this method can be removed now.